lib/cn/csched_sp3_work.c

```diff
@@ -869,7 +869,7 @@ sp3_work_wtype_length(
             list_for_each_entry(le, head, le_link) {
                 const struct kvset_stats *stats = kvset_statsp(le->le_kvset);
 
-                if (stats->kst_keys > keys_max)
+                if (stats->kst_keys > keys_max / 2)
```

Comment:
Reduced the scope of this rule as it was combining heavily compacted kvsets with uncompacted kvsets which made scatter remediation do a lot more rewriting of vblocks than is/was necessary.